daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1203,6 +1203,25 @@ object SBuiltin {
       throw DamlEUserError(args.get(0).asInstanceOf[SText].value)
   }
 
+  final case object SBToAnyTemplate extends SBuiltin(1) {
```

Comment:
Shouldn't `SB(To|From)Template` also get a comment like the primitives above have? E.g.
```
/** $error :: Text -> a */
```

Reply:
Good catch, thanks! I’ve added comments.